Makefile

```
@@ -1,5 +1,11 @@
-NAME ?= summerwind/actions-runner-controller
+ifdef USERNAME
+	NAME ?= ${USERNAME}/actions-runner-controller
+else
+	NAME ?= summerwind/actions-runner-controller
+endif
 VERSION ?= latest
+USERNAME ?= $(shell echo ${NAME} | cut -d / -f1)
```

Can we just introduce an envvar to represent the docker repo and let the user set it before running tests?

```suggestion
DOCKER_REPO ?= summerwind
NAME ?= ${DOCKER_REPO}/actions-runner-controller
VERSION ?= latest
```

---

This change breaks backwards compat for no reason.
Its not much more code to keep it compatible with the old env var, so i will keep it like it was.
If you want o can rename USERNAME to DOCKER_REPO tho.

---

Afaik a repository on dockerhub is more what NAME is, so i will rename USERNAME to DOCKER_USER instead of DOCKER_REPO, unless i get feedback on this.